distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -747,26 +747,59 @@ type Float
     to_decimal self -> Decimal = Decimal.from_float self
 
     ## ALIAS from text
-       GROUP Conversions
+       GROUP conversions
```

Comment:
```suggestion
       GROUP Conversions
```

No idea is case sensitive but was correct as `C`

Reply:
Done